ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md

```diff
@@ -0,0 +1,119 @@
+
+
+## Proposed Architecture
+
+The proposed content is as follows:
+
+***
+
+**Personas related to "Managed Services" (WIP)**
+
+**Purpose**
+To underpin many other Site Reliability Engineering-related topics, we must define the group of "personas" describing  the interaction of that person (or group of people) with an environment supported by SREs.
+
+**Service Owner**
+
+Every service needs an "owner". That is, a persona that has the primary interest in the commercial success of that service. The Service Owner takes responsibility for the success of the service and provides the strategic leadership and business context that is needed by engineers, especially during the difficult times. Without this persona, the service would not exist, and thus all other personas rely on this one. They are often referred to as the 'Champions' of this service.
+
+**Product Owner/Manager**
+
+With the strategic guidance of the Service Owner, this persona (also sometimes referred to as the Product Manager) decides on the priority of work (in this context, that may be categorized as 'feature' work or 'reliability work'). They take their inputs from many sources. Reliability, of course, is a significant input, but it is not the only input. There are many other factors which affect their decision-making. Often, they need to balance the inputs from customers and the business, with the engineering resources that are available and these demands can often be conflicting. It is the task of this role to successfully navigate all the demands, providing a path forward.
+
+Additionally, many services are dependent on other services, which have their own Product Owners. Therefore, this persona also plays a critical role in managing those dependencies, liaising with other Product Owners and Engineering resources to effectively prioritize work. Using agile concepts, this role is analogous to a 'Product Owner'.
+
+**Engineering Manager**
+
+Work needs to be done. The majority of this work is undertaken by Developers, who comprise teams led by an Engineering Manager. The primary role of this persona is to enable those teams to do the (best) work they need to do, as it is prioritized by the Product Owner. This role must also maintain working relationships with peer Engineering managers, especially where co-dependencies, shared goals and natural affinities exist.
+
+**Developers**
+
+These folks are the 'do-ers' . This persona implements the service in accordance with the 'plan' (as mostly determined by the Product Owner). They turn this plan into reality, creating and maintaining the code which underpins the service. Not only are they responsible for implementing the functionality of the service (in this context "feature work"), but they[^1] are also responsible for how the service 'operates'. Therefore, if reliability is an issue (often identified by the SREs) then they are also responsible for fixing those problems also (called "reliability work").
```

Comment:
If we're calling out folks who inform devs about reliability issues, QE, support agents, and end users (directly and also via product owners) are also frequent contributors.  Not sure it's worth listing all of these feedback channels here or not, but picking out SRE without mentioning the other major contributors feels weird.

Reply:
Software Engineers is the more current terminology.

Reply:
@wking I had that (sort of :) ) in a footnote...but that is not rendering that well. I'll probably just re-incorporate that back into the text as you suggest.

Reply:
@candita I have changed this heading to "Developers/Software Engineers".

Reply:
@wking I have re-factored this entire first paragraph to address your concerns here.